### ISSUE TITLE
Pinned apps don't show keyboard focus

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#4514]
When navigating the dashboard with the tab key, nav items and links have shading/outlines when in focus to show the current position. Pinned apps currently do not show it (although #2781 would have a tooltip pop up for each if configured), and some default indication would be good. This should be possible through hover behavior, although there  may be a more specific flag for keyboard focus if hover outlines are too distracting

**Comment by johrstrom** [comment]
Yea `:focus` is a psuedo CSS class.

**Comment by rwheirhw** [comment]
Hi is this still open?

**Comment by johrstrom** [comment]
> Hi is this still open?

I think so, yes.
